tests/preferences/test_never_remember_history.py

```diff
@@ -0,0 +1,52 @@
+import os
+import time
+
+import pytest
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.page_object import AboutPrefs
+
+
+# make sure Firefox remembers history
+@pytest.fixture()
+def add_prefs():
+    return [("browser.privatebrowsing.autostart", False)]
+
+
+def test_never_remember_history(driver: Firefox, sys_platform: str):
+    """
+    C143604: Make sure to set the pref via about:preferences, then check in about:config that the pref has been changed
+    """
+
+    AboutPrefs(driver, category="privacy").open()
+
+    # Change the settings to not remember the browser history
+    history_menulist = driver.find_element(By.ID, "historyMode")
```

Comment:
You could make this part of the Page Object Model for About:Preferences, that might make this code more reusable in the future?